packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -123,10 +122,44 @@ class LogManager {
 
   /// Logs a query, depending on the session type it will be logged directly
   /// to the database or stored in the temporary cache until the session is
-  /// closed. Call [shouldLogQuery] to check if the entry should be logged
+  /// closed. Call [_shouldLogQuery] to check if the entry should be logged
```

Comment:
Suggestion: Remove instruction that `_shouldLogQuery` should be called before this method is called.